### PR TITLE
non-archival relay

### PR DESCRIPTION
**Pull request by whyrusleeping**
real jank right now, but doesnt keep all the repo data, just does basic event validation

### CODE REVIEW

File: bgs/bgs.go [hunk 1]
```diff
@@ -1155,6 +1155,7 @@ func (bgs *BGS) handleRepoTombstone(ctx context.Context, pds *models.PDS, evt *a
 	}).Error; err != nil {
 		return err
 	}
+	u.SetTombstoned(true)
```
Comment (whyrusleeping) [inline bgs/bgs.go:1158]: this is an irrelevant fix of some other thing

File: carstore/bs.go [hunk 2]
```diff
@@ -606,7 +606,11 @@ func (ds *DeltaSession) CloseWithRoot(ctx context.Context, root cid.Cid, rev str
 	case *FileCarStore:
 		return ocs.writeNewShard(ctx, root, rev, ds.user, ds.seq, ds.blks, ds.rmcids)
 	case *NonArchivalCarstore:
-		return nil, ocs.updateLastCommit(ctx, ds.user, rev, root)
+		slice, err := blocksToCar(ctx, root, rev, ds.blks)
```
Comment (brianolson) [inline carstore/bs.go:609]: case on type should probably be a new interface method that both implement

**Comment by audreyt** [comment]
Fixed the lint error over at https://github.com/bluesky-social/indigo/pull/876

**Comment by whyrusleeping** [comment]
Thanks @audreyt !

**Pull request merged**